kinetic/distribution.yaml

```diff
@@ -1317,6 +1317,20 @@ repositories:
       url: https://github.com/ros/cmake_modules.git
       version: 0.4-devel
     status: maintained
+  cnn_bridge:
```

Comment:
Please don't add multiple ROS distributions in the same pull request.  Sometimes we have to revert changes, and since the ROS distributions are on different schedules, we sometimes have to do it asynchronously from each other.  In this case, just remove this change from this PR and open a separate one for Kinetic.

Reply:
Done